tools/schemapi/schemapi.py

```diff
@@ -477,17 +477,26 @@ def _subclasses(cls: type[Any]) -> Iterator[type[Any]]:
 
 def _todict(obj: Any, context: dict[str, Any] | None) -> Any:
```

Comment:
Is there any performance hit with `get_numpy`, `get_pandas` introduced here?

There's a whole bunch of recursion going on and those checks might add up.

Reply:
@MarcoGorelli if changing this complicates things, (saw some tests fail) don't worry.
I was just curious *if* it had a performance impact due to an earlier PR https://github.com/vega/altair/pull/3383

Reply:
interesting, thanks - locally, I'm seeing that `get_pandas` / `get_numpy` take about 4 microseconds to run. They're just doing a dictionary lookup, so it should be really fast

nonetheless, it's not much effort to define them one level up, so I can do that (and just use `sys.modules`)